### PR TITLE
BODP-5626: Ability to view seasonal services count. Timetables

### PR DESCRIPTION
**Pull request by AndreaIto**
In the Timetable data review page, as an operator, added new Sesonal Service in the blue banner.  Added new link: `Manage', this link should be taken to the seasonal service set up page, today: to the placeholder page.

Screenshot 2023-01-13 at 15 02 57

Screenshot 2023-01-13 at 15 03 13

@ITO-PaulHolland @anguskbell @marcusher-ito

### CODE REVIEW

**Comment by AndreaIto** [comment]
Screenshot 2023-01-18 at 12 52 58

File: transit_odp/organisation/querysets.py [hunk 1]
```diff
@@ -1279,3 +1279,7 @@ def add_registration_number(self) -> TSeasonalServiceQuerySet:
                 output_field=CharField(),
             )
         )
+
+    def get_seasonal_service_counter(self, org_id: int) -> int:
```
Comment (anguskbell) [inline transit_odp/organisation/querysets.py:1283]: This isn't a very descriptive name for a method on the `SeasonalService` queryset. Better would be something like `get_count_in_organisation`.

**Review by anguskbell — CHANGES_REQUESTED** [review]
If you rename the queryset method, then I'll approve.

File: transit_odp/organisation/querysets.py [hunk 2]
```diff
@@ -1279,3 +1279,7 @@ def add_registration_number(self) -> TSeasonalServiceQuerySet:
                 output_field=CharField(),
             )
         )
+
+    def get_count_in_organisation(self, org_id: int) -> int:
```
Comment (AndreaIto) [inline transit_odp/organisation/querysets.py:1283]: I renamed the method as you request `get_count_in_organisation`
@anguskbell

**Review by anguskbell — DISMISSED** [review]

**Review by anguskbell — APPROVED** [review]
If you rename the queryset method, then I'll approve.

**Pull request merged**